Form/Transformer/EntityToIdObjectTransformer.php

```diff
@@ -27,8 +28,12 @@ class EntityToIdObjectTransformer implements DataTransformerInterface
     private $om;
     private $entityName;
 
-    public function __construct(ObjectManager $om, string $entityName)
+    public function __construct($om, string $entityName)
```

Comment:
As the minimum required PHP version is 7.2, couldn't you use `object` typehint here? This would allow to remove `is_object` check. Unless that's on purpose, for some reason 🤔